XamarinCommunityToolkitSample/Resx/AppResources.resx

```diff
@@ -491,43 +491,49 @@ If the user manually assigns a row or column on a view, it will be honored.</val
     <value>Default Gravatar</value>
   </data>
-  <data name="Email" xml:space="preserve">
+    <data name="Email" xml:space="preserve">
     <value>Email</value>
   </data>
-  <data name="FullImageSource" xml:space="preserve">
+    <data name="FullImageSource" xml:space="preserve">
     <value>Full Image Source</value>
   </data>
-  <data name="GravatarSize" xml:space="preserve">
+    <data name="GravatarSize" xml:space="preserve">
     <value>GravatarSize</value>
   </data>
-  <data name="XAMLExtension" xml:space="preserve">
+    <data name="XAMLExtension" xml:space="preserve">
     <value>XAML Extension</value>
   </data>
-  <data name="UserStoppedTypingBehaviorMinimumLengthThresholdOptionLabel" xml:space="preserve">
+    <data name="UserStoppedTypingBehaviorMinimumLengthThresholdOptionLabel" xml:space="preserve">
     <value>Minimum length threshold (number of characters)</value>
   </data>
-  <data name="Effects Description" xml:space="preserve">
+    <data name="Effects Description" xml:space="preserve">
     <value>Effects are used to add visual customization on a control by control bases</value>
   </data>
-  <data name="EffectsTitle" xml:space="preserve">
+    <data name="EffectsTitle" xml:space="preserve">
     <value>Effects</value>
   </data>
-  <data name="SafeAreaShortDescription" xml:space="preserve">
-    <value>The SafeAreaEffect is an effectwill help to make sure that content isn't clipped by rounded device corners, the home indicator, or the sensor housing on an iPhone X.</value>
+    <data name="SafeAreaShortDescription" xml:space="preserve">
+    <value>The SafeAreaEffect is an effect that will help to make sure that content isn't clipped by rounded device corners, the home indicator, or the sensor housing on an iPhone X.</value>
   </data>
-  <data name="SafeAreaLongDescription" xml:space="preserve">
+    <data name="SafeAreaLongDescription" xml:space="preserve">
     <value>The SafeAreaEffect is an effect that can be added to any element through an attached property to indicate whether or not that element should take current safe areas into account. This is an area of the screen that is safe for all devices that use iOS 11 and greater. Specifically, it will help to make sure that content isn't clipped by rounded device corners, the home indicator, or the sensor housing on an iPhone X. The effect only targets iOS, meaning that on other platforms it does not do anything.</value>
   </data>
-  <data name="ToggleSafeAreaEffect" xml:space="preserve">
+    <data name="ToggleSafeAreaEffect" xml:space="preserve">
     <value>Toggle SafeAreaEffect</value>
   </data>
+    <data name="StateLayoutTitle" xml:space="preserve">
+    <value>StateLayout</value>
+  </data>
+    <data name="StateLayoutDescription" xml:space="preserve">
+    <value>A collection of attached properties that let you specify one or more state views for any of your existing layouts.</value>
+  </data>
 </root>
```

Comment:
Hmm, this file updated indentation...

Reply:
Yeah, seems my VSMac save messes it up. Tried to fix.